### PR TITLE
Pull changes

### PR DESCRIPTION
**Pull request by AGegen**

### CODE REVIEW

**Review by iteles — COMMENTED** [review]
@AGegen Did you mean to open this PR to your fork rather than to this tutorial?

File: index.html [hunk 1]
```diff
@@ -5,17 +5,20 @@
...
-  text-align: center;
-  font-family: sans-serif;
-}
-</style>
+    <div style="background-color:#f0f0f5;color:#006699;padding:20px;">
+    <p>
+        Dear employee,
+    </p>
+    <p>
+    You have been chosen to provide feedback for you manager. Please answer the survey questions honestly.
```
Comment (nelsonic) [inline index.html:13]: @AGegen stoked that you are learning using this tutorial. 🎉 
but this is _probably_ not relevant to the other people using this...? 🤔

Comment (AGegen) [reply index.html:13]: Oh hi, yes I didn't mean to do that. I'm sorry. Thank you for the great tutorial!

**Comment by nelsonic** [comment]
Closing PR as unintentionally opened. see: https://github.com/dwyl/learn-heroku/pull/28#discussion_r239208654

**Pull request closed without merging**